content/content-strategy/define-user-content-needs/create-actionable-artefacts/user-job-stories/text3.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: text/textblock
+---
+### [1]Step 6: Write content to match your user stories
```

Comment:
Incorrect display level. Needs to be `### [2]` to show a heading 3 and be consistent with the other steps listed on the page.

Reply:
Steve fixed

Reply:
Fixed